jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -73,6 +73,7 @@ public class ErrorHandler implements Request.Handler
     boolean _showStacks = false;
     boolean _showCauses = false;
     boolean _showMessageInTitle = true;
+    int _bufferSize = -1;
```

Comment:
Hmmm why are these fields not private?   Not a problem for this PR, but strange....